scilpy/viz/sampling_scheme.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import division
+
+import numpy as np
+from tempfile import mkstemp
+
+from dipy.data import get_sphere
+from fury import actor, window
+
+# TODO: Make it robust to more than 10 b-values
```

Comment:
Also here TODO, maybe create an issue?

Reply:
I kept the same behavior (same color if 10 b-values or lower) but added creation of a color map if more than 10 b-values